be/src/exprs/vectorized/runtime_filter.h

```diff
@@ -204,6 +204,7 @@ class JoinRuntimeFilter {
         bool use_merged_selection;
         std::vector<uint32_t> hash_values;
         const std::vector<int32_t>* bucketseq_to_partition;
+        bool compatibility;
```

Comment:
Why the name is compatibility?

Reply:
We need to be compatible with grayscale upgrades, and give a variable to indicate if all of BE are new versions.
Do you have a better idea ?